cmd/service-controller/sites.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/skupperproject/skupper/client"
+)
+
+const (
+	SiteManagement string = "SiteManagement"
+)
+
+type SiteManager struct {
+	cli *client.VanClient
+}
+
+func newSiteManager(cli *client.VanClient) *SiteManager {
+	return &SiteManager{
+		cli: cli,
+	}
+}
+
+func (m *SiteManager) revokeAccess() error {
+	err := m.cli.RevokeAccess(context.Background())
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func serveSites(m *SiteManager) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		if r.Method == http.MethodDelete {
```

Comment:
I would prefer POST as the method for triggering this action (as to me it is not really deleting an exposed resource).

Reply:
Changed to `POST`